cwl/src/main/scala/cwl/CwlWomExpression.scala

```diff
@@ -17,6 +19,23 @@ sealed trait CwlWomExpression extends WomExpression {
   override def evaluateType(inputTypes: Map[String, WomType]): ErrorOr[WomType] = cwlExpressionType.validNel
 }
 
+case class ResourceRequirementExpression(expression: Expression,
```

Comment:
ToL: This seems like it could be "JobPreparationExpression", since AFAICT it doesn't do anything resource-specific